src/app/home-page/home-logged-in/home-logged-in.component.html

```diff
@@ -64,9 +64,9 @@ <h3>Recent Activity</h3>
         <div class="w-100">
           <h3>Links</h3>
           <ul>
-            <li><a routerLink="quick-start">Quickstart</a></li>
-            <li><a routerLink="accounts">My Account</a></li>
-            <li><a routerLink="starred">My Starred</a></li>
+            <li><a [routerLink]="['/quick-start']">Quickstart</a></li>
+            <li><a [routerLink]="['/accounts']">My Account</a></li>
+            <li><a [routerLink]="['/starred']">My Starred</a></li>
```

Comment:
No need to change, but for future reference, I think you can just assign the property to a string literal like so:
`<a routerLink="/quick-start">`

Reply:
Ooh okay got it! :+1: